header.php

```diff
@@ -7,17 +7,31 @@
     <?php wp_head(); ?>
 </head>
 <body <?php body_class(); ?>>
-<header>
-    <div class="container">
-        <h1><a href="<?php echo esc_url(home_url('/')); ?>"><?php bloginfo('name'); ?></a></h1>
-        <nav>
-            <?php
-            wp_nav_menu(array(
-                'theme_location' => 'primary-menu',
-                'container' => false,
-                'menu_class' => 'nav-menu',
-            ));
-            ?>
-        </nav>
-    </div>
-</header>
\ No newline at end of file
+    <header class="header">
+        <div class="header__container">
+            <a href="<?php echo esc_url(home_url('/')); ?>" class="header__branding">
+                EZEBLaMER
+            </a>
+            <nav class="header__navigation">
+                <?php
+                $current_page = is_front_page() ? 'inicio' : '';
+                ?>
+                <a href="<?php echo esc_url(home_url('/')); ?>" 
+                   class="nav-link <?php echo $current_page === 'inicio' ? 'active' : ''; ?>">
+                    Inicio
+                </a>
```

Comment:
- Solo veo que la usas aqui, asi que puedes hacerlo sin crear una variable
- En el archivo anterior usas la funcion de traduccion, asi que no se por que noo sigues usandola. No se si solo copiaste sin saber que es
- No se si ya vas por esa parte, pero se puede construir el menu en el admin y en el tema hacer el `for` para recorrer los item. Igual ya llegaras a eso en tu estudio (espero)
```suggestion
                <a href="<?php echo esc_url(home_url('/')); ?>" 
                   class="nav-link <?php echo is_front_page() ? 'active' : ''; ?>">
                    <? echo __('Inicio', 'aprendiendoezequiel'); ?>
                </a>
```